app/javascript/components/key-performance-indicators/components/stacked-percentage-bar/container.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import makeStyles from "@material-ui/styles/makeStyles";
+
+import styles from "./styles.css";
+import StackedPercentageBarMeter from "./stacked-percentage-bar-meter";
+import StackedPercentageBarLabel from "./stacked-percentage-bar-label";
+
+const StackedPercentageBar = ({ percentages, className }) => {
+  const css = makeStyles(styles)();
+
+  if (percentages.length > 2)
+    throw "StackedPercentageBar components only support a max of 2 percentages";
```

Comment:
Does this component fail if this condition met?

Reply:
Unfortunately, yes. all of the colors for percentage meters are manual styles so we'd need a dynamic way to pick colors for sets of percentages > 2.